Lib/_pydecimal.py

```diff
@@ -2778,12 +2753,8 @@ def sqrt(self, context=None):
 
         # find n = floor(sqrt(c)) using Newton's method
         n = 10**prec
-        while True:
-            q = c//n
-            if n <= q:
-                break
-            else:
-                n = n + q >> 1
+        while (q := c//n) < n:
```

Comment:
This is where you went too far :)
*flies away*

Reply:
What?  I think this is the change with the most win in the PR!

Reply:
I agree with @phord on this one. 

The places replacing the whole `while: True` followed by an assignment and a conditional break looks much cleaner to me now. 

I was slightly sceptic at first but I’m warming up to this proposal now.